mama/c_cpp/src/c/mama/publisher.h

```diff
@@ -154,8 +232,8 @@ mamaPublisher_sendFromInbox (mamaPublisher publisher,
 MAMAExpDLL
 extern mama_status
 mamaPublisher_sendReplyToInboxHandle (mamaPublisher publisher,
-                                	  mamaMsgReply replyAddress,
-                                	  mamaMsg reply);
+                                      mamaMsgReply replyAddress,
```

Comment:
Same as above this is for parameter alignment. I know if can be painful because a lot of these files contain mixed-standards but the double-column approach is the one we try and use for any new code where all arguments won't fit on a single line (or optionally even if they would).